saleor/urls.py

```diff
@@ -57,3 +57,14 @@
     urlpatterns += [
         url(r'^static/(?P<path>.*)$', serve)] + static(
             settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+
+if settings.ENABLE_DEBUG_TOOLBAR:
+    import debug_toolbar
+    urlpatterns += [
+        url(r'^__debug__/', include(debug_toolbar.urls)),
+    ]
```

Comment:
We could move hanging brackets one line up. It's preferred style in our codebase.